ee/clickhouse/queries/paths/paths.py

```diff
@@ -205,13 +214,17 @@ def get_session_threshold_clause(self) -> str:
 
         return "arraySplit(x -> if(x.3 < %(session_time_threshold)s, 0, 1), paths_tuple)"
 
-    def get_target_clause(self) -> Tuple[str, Dict]:
+    def get_target_clause(self) -> Tuple[str, str, Dict]:
         params: Dict[str, Union[str, None]] = {"target_point": None, "secondary_target_point": None}
 
         if self._filter.end_point and self._filter.start_point:
             params.update({"target_point": self._filter.end_point, "secondary_target_point": self._filter.start_point})
             return (
                 """
+            , indexOf(paths, %(secondary_target_point)s) as start_target_index
+            , indexOf(paths, %(target_point)s) as end_target_index
+            """,
+                """
             , indexOf(compact_path, %(secondary_target_point)s) as start_target_index
```

Comment:
I assume you intended to remove this form here, and use compact_path above?

Reply:
Nvm, looks alright

Reply:
Edit: This becomes problematic with wildcard start & end points? And path persons with the same